packages/server/graphql/types/AzureDevOpsRemoteProject.ts

```diff
@@ -0,0 +1,53 @@
+import {GraphQLBoolean, GraphQLID, GraphQLNonNull, GraphQLObjectType, GraphQLString} from 'graphql'
+import AzureDevOpsProjectId from 'parabol-client/shared/gqlIds/AzureDevOpsProjectId'
+import {GQLContext} from '../graphql'
+import IntegrationProviderServiceEnum from './IntegrationProviderServiceEnum'
+import RepoIntegration, {repoIntegrationFields} from './RepoIntegration'
+
+const AzureDevOpsRemoteProject = new GraphQLObjectType<any, GQLContext>({
+  name: 'AzureDevOpsRemoteProject',
+  description: 'A project fetched from Azure DevOps in real time',
+  interfaces: () => [RepoIntegration],
+  isTypeOf: ({service}) => service === 'azuredevops',
+  fields: () => ({
+    ...repoIntegrationFields(),
+    id: {
+      type: new GraphQLNonNull(GraphQLID),
+      resolve: ({instanceId, key}) => AzureDevOpsProjectId.join(instanceId, key)
+    },
+    service: {
+      type: new GraphQLNonNull(IntegrationProviderServiceEnum),
+      resolve: () => 'azuredevops'
+    },
+    teamId: {
+      type: new GraphQLNonNull(GraphQLID),
+      description: 'The parabol teamId this issue was fetched for'
+    },
+    userId: {
+      type: new GraphQLNonNull(GraphQLID),
+      description: 'The parabol userId this issue was fetched for'
+    },
+    self: {
+      type: new GraphQLNonNull(GraphQLID)
+    },
+    instanceId: {
+      type: new GraphQLNonNull(GraphQLID),
+      description:
+        'The instance ID that the project lives on. Does not exist on the Azure DevOps object!'
```

Comment:
-1 

> Does not exist on the Azure DevOps object!

This part is only confusing. Does that mean this field is broken and needs a `resolve` function? Do we need this field then?

Reply:
That comment was carry over from the JiraRemoteProject.  Removed